classes/site/Version.inc.php

```diff
@@ -284,6 +284,23 @@ function getVersionString($numeric = true) {
 
 		return $numericVersion;
 	}
+
+	/**
+	 * Checks if the Version is compatible with the given string of constraints for the version, 
+	 * formatted per composer/semver specifications; 
+	 * c.f. https://getcomposer.org/doc/articles/versions.md#writing-version-constraints
+	 * Returns:
+	 * 		true iff the version given is compatible with this version
+	 * 		false iff the version given is incompatible with this version
+	 * @param $version string the string of constraints for the version to be checked against
+	 * @return boolean
+	 */
+	function isCompatible($version) {
```

Comment:
Ok, this is pedantic, but can you rename the parameter to `$constraints`, so the current version can just be `$version` rather than `$systemVersion`, which is a bit of a misnomer?